### DISCUSSION TITLE
Guidelines for Making Pull Requests

### DISCUSSION THREAD

**Discussion by Amna-Hassan04** [Amna-Hassan04/Serenity-Guide#15] [Announcements]
## Contribution Guidelines

Welcome to the project! Please follow these guidelines to ensure a smooth collaboration:

1. **Add Your Profile**  
   Once you've completed a task, please add your name and details as a contributor in the project's **Contributors Table/List**.

2. **Showcase Your Work**  
   After completing your task, kindly attach any relevant **screenshots or videos** that demonstrate the functionality of your contribution. This helps maintain transparency and verifies the completion of tasks.

3. **Task Assignment Process**  
   - All issues or tasks will be assigned on a **first-come, first-served** basis.
   - If someone has already requested assignment to a task, please **do not spam** by requesting the same task. Respect the queue and wait for new opportunities.

4. **Propose New Ideas**  
   If you have suggestions or new features you'd like to implement, first discuss them in the **"Ideas" channel** in the project's Discussions tab. This ensures alignment with the project's goals and avoids duplicate work.

**Comment by ghost** [comment]
Thank you for the opportunity. Will follow them

**Comment by sid7219** [comment]
thank you for this great opportunity.
